topics/microbiome/tutorials/dada2/tutorial.md

```diff
@@ -0,0 +1,884 @@
+objectives:
+- The learning objectives are the goals of the tutorial
+- They will be informed by your audience and will communicate to them and to yourself
+  what you should focus on during the course
+- They are single sentences describing what a learner should be able to do once they
+  have completed the tutorial
+- You can use Bloom's Taxonomy to write effective learning objectives
+time_estimation: 3H
+key_points:
+- The take-home messages
+- They will appear at the end of the tutorial
+subtopic: metabarcoding
+tags:
+- metabarcoding
+- 16S
+- microgalaxy
+contributions:
+   authorship:
+    - bebatut
+edam_ontology:
+- topic_3697 # Microbial ecology
+- topic_0637 # Taxonomy
+- topic_0080 # Sequence analysis
+- topic_4038 # Metabarcoding
+
+---
+
+> <comment-title>Sources</comment-title>
+>
+> This tutorial is significantly based on [DADA2 Pipeline Tutorial](https://benjjneb.github.io/dada2/tutorial.html) and ["Introduction to dada2"](https://bioconductor.org/packages/devel/bioc/vignettes/dada2/inst/doc/dada2-intro.html)
+>
+{: .comment}
+
+The investigation of environmental microbial communities and microbiomes has been revolutionized by the 
+development of high-throughput amplicon sequencing. In amplicon sequencing a particular genetic locus, 
+for example the 16S rRNA gene in bacteria, is amplified from DNA extracted from the community of interest, 
+and then sequenced on a next-generation sequencing platform. This technique removes the need to culture 
+microbes in order to detect their presence, and cost-effectively provides a deep census of a microbial community.
+
+{% include topics/microbiome/tutorials/mothur-miseq-sop/background_16S.md %}
+
+To process 16S amplicon sequencing data, several tools exist: **QIIME2** ({% cite Bolyen_2019 %}), **Mothur** ({% cite Schloss_2009 %}), **Lotus2** ({% cite _zkurt_2022 %}), and **DADA2** ({% cite Callahan_2016 %}). All these tools can be used in Galaxy. In this tutorial we will use **DADA2**.
+
+> <comment-title>Processing 16S amplicon data in Galaxy using other tools than DADA2</comment-title>
+>
+> Tutorials are available for the other tools in Galaxy:
+> - [QIIME 2 Cancer Microbiome Intervention](https://docs.qiime2.org/jupyterbooks/cancer-microbiome-intervention-tutorial/index.html#)
+> - [QIIME 2 Moving Pictures](https://docs.qiime2.org/2024.2/tutorials/moving-pictures-usage)
+> - [16S Microbial Analysis with mothur]({% link topics/microbiome/tutorials/mothur-miseq-sop/tutorial.md %})
+>
+{: .comment}
+
+The output of the dada2 pipeline is a feature table of amplicon sequence variants (an ASV table): A matrix with rows corresponding to samples and columns to ASVs, in which the value of each entry is the number of times that ASV was observed in that sample. This table is analogous to the traditional OTU table, except at higher resolution, i.e exact amplicon sequence variants rather than (usually 97%) clusters of sequencing reads.
```

Comment:
Isn't it: 

- rows = ASVs
- columns = Samples

Reply:
Maybe instead of the last sentence:

```
This table is analogous to the traditional OTU table, except that one gets the actual sequences of the "true" sequence variants instead of rather anonymous and abstract entities called OTU1, OTU2, etc. 
```

Reply:
Nice catch and suggestion. Thanks